backend/src/pong/game/game.module.ts

```diff
@@ -8,5 +8,6 @@ import { Game } from './entities/game.entity';
 	imports: [TypeOrmModule.forFeature([Game])],
 	controllers: [GameController],
 	providers: [GameService],
+	exports: [GameService],
```

Comment:
Did you figure out what exports from a module does? It's still vague to me.. so if you know and can talk me trough it a bit that would be greeat :P

Reply:
Yes, let's have a chat about it!